src/Data/User/Pisa/pisaUserDataClass.ts

```diff
@@ -1,24 +1,16 @@
-import { provideDataClass } from 'scrivito'
+import { DataConnectionError, provideDataClass } from 'scrivito'
 import { pisaClient } from '../../pisaClient'
-import { toClientParams } from '../../toClientParams'
-import { DataIndexResponse } from '../../types'
 import { postProcessUserData } from '../UserDataClass'
 
 export async function pisaUserDataClass() {
   const userClient = await pisaClient('user')
 
   return provideDataClass('User', {
     connection: {
-      index: async (params) => {
-        const { results, continuation, count } = await (userClient.get('', {
-          params: toClientParams(params),
-        }) as Promise<DataIndexResponse>)
-
-        return {
-          results: await Promise.all(results.map(postProcessUserData)),
-          continuation,
-          count,
-        }
+      index: async () => {
+        throw new DataConnectionError(
+          'Pisa does not support "index" for users due to security considerations.',
+        )
```

Comment:
* [ ] I tried this out with a data list page and a data group with a list. I only see "Editor note: Data is empty.". Is this message supposed to be visible? Where? If it is:
* [ ] Pisa -> PisaSales, but:
* [ ] I think an editor will not understand what `"index" for users` means. Can we rephrase this in more explanatory way? Something along the lines of "Listing users is not supported due to data protection reasons."

Reply:
> I only see "Editor note: Data is empty.". Is this message supposed to be visible? Where?

With the current SDK version this message is not visible. But it will become visible with the next version of the SDK again (see https://github.com/Scrivito/scrivito-portal-app/pull/431).